src/test/operators/table_scan_string_test.cpp

```diff
@@ -147,14 +147,14 @@ TEST_P(OperatorsTableScanStringTest, ScanLikeEmptyStringDict) {
   EXPECT_TABLE_EQ_UNORDERED(scan->get_output(), expected_result);
 }
 
-TEST_P(OperatorsTableScanStringTest, ScanLikeStartingOnDictCxlumn) {
+TEST_P(OperatorsTableScanStringTest, ScanLikeStartingOnDictSegment) {
```

Comment:
I wasn't sure about these, because some of these tests have multiple dict segments. It is more a "..OnAColumnWhereSegmentsAreCommonlyEncodedAsDictSegments" :D 

In the end, the test names don't matter too much, so either way works.

Reply:
I agree, I was thinking about that, too.
My decision was to use column for data types (e.g. FloatColumn) and segment for encoding schemes (e.g. DictSegment).